common/locales/en/tasks.json

```diff
@@ -89,7 +89,7 @@
     "sureDelete": "Are you sure you want to delete this task?",
     "streakCoins": "Streak Bonus!",
     "pushTaskToTop": "Push task to top",
-    "pushTaskToBottom": "Push task to bottom",
+    "pushTaskToBottom": "Push task to bottom. Hold ctrl or cmd to push to top.",
```

Comment:
It looks like you've got this backward. Push to top is still the default, and Ctrl or Cmd flips it to push-to-bottom. It's the `pushTaskToBottom` string that doesn't get used:
![image](https://cloud.githubusercontent.com/assets/4501321/11428508/c9c4bd3e-9431-11e5-91a6-98738cb8c9b4.png)